### PR TITLE
Tune coco parameters

### PR DESCRIPTION
**Pull request by Flova**
## Proposed changes
- tune hyperparams
- scale based on mini batch size
- linear instead of step wise decay
- use sgd
- only log once per optimizer step

## Related issues
- #818 
- #820

## Necessary checks
- [ ] Update poetry package version [semantically](https://semver.org/)
- [x] Write documentation
- [x] Create issues for future work
- [x] Test on your machine

### CODE REVIEW

**Comment by J-LINC** [comment]
> hi ，Regarding what you mentioned:linear instead of step wise decay, I have reviewed your code and found that there is only one learning rate decay strategy, which is to adjust according to steps. How can I perform linear decay or other decay methods
as for using sgd ,i think you are right for most trains and i successed when i using sgd in trianing bdd100k
